### PR TITLE
point config to v0.6 branch head and add deployment trouble shooting instruction

### PR DESCRIPTION
**Pull request by kunmingg**
- Point config to v0.6 branch head 
- Publish trouble shooting instruction for kubeflow deployment.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/975)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
/assign

Preview: https://deploy-preview-975--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/

**Comment by sarahmaddox** [comment]
Thanks for adding this info @kunmingg! Before I do a review of the content, please would you move the content to the following page:
https://www.kubeflow.org/docs/gke/troubleshooting-gke/

**Comment by sarahmaddox** [comment]
Friendly ping @kunmingg

**Comment by sarahmaddox** [comment]
@kunmingg Please would you take a look at my review comment.

**Comment by sarahmaddox** [comment]
@kunmingg Would you like me to take over this PR? I can copy your content into the correct page, then send you a new PR for review.

**Comment by kunmingg** [comment]
@sarahmaddox 
Sorry for late reply, PR updated.

File: content/docs/gke/troubleshooting-gke.md [hunk 1]
```diff
@@ -19,10 +19,18 @@ For more help, try the
 
 Here are some tips for troubleshooting Cloud IAP.
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
+* Make sure you are using HTTPS.
+* Check project [quota page](https://pantheon.corp.google.com/iam-admin/quotas) to see if there are any quota limit, increase them as needed.
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: Replace the internal "pantheon" link with the external console link.

Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: s/limit/limits

File: content/docs/gke/troubleshooting-gke.md [hunk 2]
```diff
@@ -19,10 +19,18 @@ For more help, try the
...
+* Check [deployment manager page](https://console.cloud.google.com/deployments) and see if there’s a failed deployment.
+* Check if endpoint is up: do [DNS lookup](https://mxtoolbox.com/DNSLookup.aspx) against your IAP url and see if can resolve to correct ip.
+* Check if certificate succeeded: “kubectl describe certificates -n kubeflow” should give you certificate status.
+* Make sure you added https://<deployment>.endpoints.<project>.cloud.goog/_gcp_gatekeeper/authenticate
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: Add a link to the doc that describes how to do that: /docs/gke/deploy/oauth-setup/

File: content/docs/gke/troubleshooting-gke.md (hunk 1 above)
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: I'm not sure what this bit means: "to see if there are any quota limit". I guess there are always quota limits. Does it mean that the console will display info about exceeded quota?

File: content/docs/gke/troubleshooting-gke.md [hunk 3]
```diff
@@ -19,10 +19,18 @@ For more help, try the
...
 Here are some tips for troubleshooting Cloud IAP.
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
+* Make sure you are using HTTPS.
+* Check project [quota page](https://pantheon.corp.google.com/iam-admin/quotas) to see if there are any quota limit, increase them as needed.
+* Check [deployment manager page](https://console.cloud.google.com/deployments) and see if there’s a failed deployment.
+* Check if endpoint is up: do [DNS lookup](https://mxtoolbox.com/DNSLookup.aspx) against your IAP url and see if can resolve to correct ip.
+* Check if certificate succeeded: “kubectl describe certificates -n kubeflow” should give you certificate status.
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: Use code formatting: 

```
`kubectl describe certificates -n kubeflow`
```

File: content/docs/gke/troubleshooting-gke.md (hunk 2 above)
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: I think you need to put the URL in code formatting, otherwise it will be rendered on the webpage.

**Comment by sarahmaddox** [comment]
Preview:
https://deploy-preview-975--competent-brattain-de2d6d.netlify.com/docs/gke/troubleshooting-gke/

(but preview is currently not available as tests seem to be stuck)

File: content/docs/gke/troubleshooting-gke.md [hunk 4]
```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
 For more help, try the 
 [general Kubeflow troubleshooting guide](/docs/other-guides/troubleshooting).
 
-## Troubleshooting Cloud Identity-Aware Proxy (Cloud IAP)
+## Troubleshooting kubeflow deployment on GCP
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: s/kubeflow/Kubeflow

File: content/docs/gke/troubleshooting-gke.md [hunk 5]
```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
...
 
 Here are some tips for troubleshooting Cloud IAP.
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: Change "Cloud IAP" to "GCP" to match change in heading.

File: content/docs/gke/troubleshooting-gke.md [hunk 6]
```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
...
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: Add a full stop at the end of the line, for consistency with other bullet points.

File: content/docs/gke/troubleshooting-gke.md [hunk 7]
```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
...
+* Make sure you are using HTTPS.
+* Check project [quota page](https://console.cloud.google.com/iam-admin/quotas) to see if any service's current usage reached quota limit, increase them as needed.
+* Check [deployment manager page](https://console.cloud.google.com/deployments) and see if there’s a failed deployment.
+* Check if endpoint is up: do [DNS lookup](https://mxtoolbox.com/DNSLookup.aspx) against your IAP url and see if can resolve to correct ip.
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: s/url/URL

Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: On first usage of "IAP", provide full term:

"Cloud Identity-Aware Proxy (Cloud IAP)"

Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: s/ip/IP address

File: content/docs/gke/troubleshooting-gke.md [hunk 8]
```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
...
+* Check if certificate succeeded: `kubectl describe certificates -n istio-system` should give you certificate status.
+* Check ingress status: `kubectl describe ingress -n istio-system`
+* Check if [endpoint entry](https://console.cloud.google.com/endpoints) is created. There should be one entry with name `<deployment>.endpoints.<project>.cloud.goog`
+  * If endpoint entry doesn't exist, check `kubectl describe cloudendpoint -n istio-system`
+* If using IAP: make sure you [added](/docs/gke/deploy/oauth-setup/) `https://<deployment>.endpoints.<project>.cloud.goog/_gcp_gatekeeper/authenticate`
+as an authorized redirect URI for the OAUTH credentials used to create the deployment.
+* If using IAP: see the guide to 
   [monitoring your Cloud IAP setup](/docs/gke/deploy/monitor-iap-setup/).
 * See the sections below for troubleshooting specific problems.
+* Please [report bug](https://github.com/kubeflow/kubeflow/issues/new?template=bug_report.md) if all above items look good.
```
Comment (sarahmaddox) [inline content/docs/gke/troubleshooting-gke.md]: Suggestion: Replace this:
"if all above items look good"
with this:
"if you can't the problem by following the above steps"

Reason: On first reading, it sounds a little strange to say they should report a bug if everything looks good.

**Comment by sarahmaddox** [comment]
I'll let this PR through as it contains useful information, and we'll fix up the items mentioned in the comments in a later PR. Thanks for the update!

**Pull request merged**